#This message is used to send command to Herkulex
# Angle(degree) Move Command

int8 model_num #0:DRS-0101, 1:DRS-0102, 2:DRS-0201, 3:DRS-0401, 4:DRS-0402, 5:DRS-0601, 6:DRS-0602 
int8 id
int8 led
int16 playtime
float64 target_angle
bool unit # true: degree / false: radian
int16 jogmode
---
string command 
bool command_result
